.github/workflows/ci.yml

```diff
@@ -51,10 +51,10 @@ jobs:
           python -m pip install --upgrade pip setuptools wheel
           python -m pip install --upgrade --upgrade-strategy eager -r requirements/dev.txt
           python -m pip install --upgrade --upgrade-strategy eager -e .
-          if [ "$RUNNER_OS" != "Windows" ] && [ ${{ matrix.python }} < '3.10' ]; then  # Skip Prophet tests on Windows as installation complex. Skip on Python 3.10 as not supported.
+          if [ "$RUNNER_OS" != "Windows" ] && ${{ matrix.python-version != '3.10' }}; then  # Skip Prophet tests on Windows as installation complex. Skip on Python 3.10 as not supported.
```

Comment:
This seems pretty silly, so when `3.11` comes out we will need to add more `!=`. Is this a documented shortcoming of the Github actions expression language?

Reply:
Indeed, it isn't a long-term solution. There are a few issues that mention this to some degree e.g. https://github.com/actions/setup-python/issues/160 and https://github.com/actions/runner/issues/1989. The proposed solution is to contain `3.10` in quotes so it is not parsed as a float, but this is no good for version comparisons.

Long term I am hoping the github expression language builds in support for it, or even better we can simply remove this conditional because `prophet` improves support for newer python versions (tbh if they don't, we will have to think about removing it as a dep anyway I think?). We could also write some custom bash code to do the version comparison but I didn't want to over-complicate now. I couldn't find any more solutions but can also do some more research!

Reply:
Was more of a discussion point, happy to leave as is now.